README.md

```diff
@@ -7,3 +7,11 @@ this is a todo list with javascript
 ### desktop
 
 ![desktop](./img/website-img.png)
+
```

Comment:
- [ ] please add more details to your readme file and a live demo link

Reply:
hello Zerradi 

did you try to open it from webpack server ? because it's working on my pc